### ISSUE TITLE
Download?

### ISSUE THREAD

**Issue by spannerman79** [Alastors/thaumicinsurgence#1]
Do you have a curseforge page?
If not - you should put your releases onto https://github.com/Alastors/thaumicinsurgence/releases

**Comment by Alastors** [comment]
> Do you have a curseforge page? If not - you should put your releases onto https://github.com/Alastors/thaumicinsurgence/releases

Yeah actually! Sorry, I've been bad at pulling the latest here, give me a moment, I'm pulling from the main branch right now!

(I know that this is the root branch, but I've been doing a lot of work on the branch specific to the GTNH pack that I kept forgetting to pull here...)

**Comment by Alastors** [comment]
But my mod actually just got approved on curseforge a couple days ago so that will be up soon as well!

**Comment by Alastors** [comment]
@spannerman79 
Here's the actual link to my curseforge page!
https://www.curseforge.com/minecraft/mc-mods/thaumic-insurgence

And since it may take a day or two for the latest version to get pushed to curseforge, so here's a zip with the mod, otherwise you can download it from releases now!
[thaumicinsurgence-1.7.10-0.0.0.2.jar.zip](https://github.com/Alastors/thaumicinsurgence/files/9464834/thaumicinsurgence-1.7.10-0.0.0.2.jar.zip)

(the item has been made faster and is actually craftable now since it's fully functional!!! Thank you for your interest in thaumic insurgence!)
